### PR TITLE
Social Anti-RightClick: fmkorea.com

### PR DESCRIPTION
**Pull request by ghost**
### URLs
https://www.fmkorea.com/best2/3727941251
https://www.fmkorea.com/best2/3726267737

### CODE REVIEW

**Comment by piquark6046** [comment]
Related PR: #86083

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1253,6 +1253,7 @@ imslp.org#%#AG_onLoad(function(){var b=document.getElementById("sm_dl_wait");if(
 ||cdn.web.tv/asset/javascripts/smothScroll.min.js
 ||serialuk.net/js/copyright.js
 !--- CSS ---
+fmkorea.com#$##bd_capture { background: none!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1256]: Why do you need this rule?

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1256]: To remove the `div` property displaying your IP address or user ID.

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1256]: Example: https://www.fmkorea.com/best2/3726267737

**Review by zloyden — APPROVED** [review]

**Pull request merged**